packages/edgeidentity/ios/src/RCTAEPEdgeIdentity.m

```diff
@@ -27,7 +30,12 @@ - (dispatch_queue_t)methodQueue
 
 RCT_EXPORT_METHOD(getExperienceCloudId:(RCTPromiseResolveBlock) resolve rejecter:(RCTPromiseRejectBlock)reject) {
     [AEPMobileEdgeIdentity getExperienceCloudId:^(NSString * _Nullable experienceCloudId, NSError * _Nullable error) {
-        resolve(experienceCloudId);
+        if (error) {
+            reject(EXTENSION_NAME, FAILED_TO_CONVERT_EVENT_MESSAGE, nil);
```

Comment:
Instead of returning FAILED_TO_CONVERT_EVENT_MESSAGE, can we read the AEPError message from error?

Reply:
Change to write it as this way right now
reject(EXTENSION_NAME, nil, error);
Sample output:
WARN  Possible Unhandled Promise Rejection (id: 0):
Error: The operation couldn’t be completed. (AEPCore.AEPError error 1.)

-----------------
If use 
reject(EXTENSION_NAME, FAILED_TO_CONVERT_EVENT_MESSAGE, error);

Sample output  would be generic
WARN  Possible Unhandled Promise Rejection (id: 0):
Error: Failed to get Experience Cloud ID

Let me know what you think